tests/test_modules/test_dataset_analysis/test_syntax_tagging.py

```diff
@@ -57,3 +56,53 @@ def test_syntax_tagging(simple_text_config):
     # Tags should changed for utterances below, based on new config values
     assert not json_output_2[2].tags[SmartTag.short] and json_output_2[0].tags[SmartTag.long]
     assert not json_output_2[3].tags[SmartTag.short] and json_output_2[0].tags[SmartTag.long]
+
+
+def test_syntax_tagging_french(simple_text_config_french):
+    mod = SyntaxTaggingModule(
+        DatasetSplitName.eval,
+        simple_text_config_french,
+    )
+
+    assert mod is not None
+    batch = {
+        DatasetColumn.idx: [0, 1, 2, 3],
+        "utterance": [
+            "adore les biscuits!",
+            "c'est terrible. C'est horrible pour moi d'écrire ce test, mais je m'amuse bien.",
+            f"J{chr(8217)}aimerais aller",  # Single quote; sm model struggled with j'aimerais subj
+            "le sucre et les biscuits!",  # As currently implemented, no subject or object
+            "Indiquez-moi l'état de mes demandes",  # First word (a) capitalized and (b) verb
+        ],
+        "label": [0, 1, 0, 1, 0],
+    }
+    json_output = mod.compute(batch)
+    assert len(json_output) == 5
+
+    assert json_output[0].tags[SmartTag.no_subj]
+    assert not any([json_output[0].tags[SmartTag.no_obj], json_output[0].tags[SmartTag.no_verb]])
+
+    assert not any(
+        [
+            json_output[1].tags[SmartTag.no_subj],
+            json_output[1].tags[SmartTag.no_verb],
+            json_output[1].tags[SmartTag.no_verb],
+        ]
+    )
```

Comment:
I don't see a huge advantage using `any()` instead of the following, which might even be easier to read/parse. That's a detail, though, so as you wish.
```suggestion
    assert not json_output[1].tags[SmartTag.no_subj]
    assert not json_output[1].tags[SmartTag.no_verb]
    assert not json_output[1].tags[SmartTag.no_verb]
```